priv/sites/zotonic_status/modules/mod_zotonic_site_management/support/zotonic_status_addsite.erl

```diff
@@ -192,6 +199,7 @@ create_gitignore(SiteDir) ->
     file:write_file(filename:join([SiteDir, ".gitignore"]), GitIgnore).
 
 
+-spec copy_skeleton_dir(any(), any(), list(), #context{}) -> ok | {error, Reason :: binary()}.
```

Comment:
The spec in line 202 uses a record, please define a type for the record and use that instead.